runtime/near-vm-runner-standalone/Cargo.toml

```diff
@@ -22,8 +22,7 @@ to make sure it has expected behavior once deployed to the blockchain.
 [dependencies]
 serde = { version = "1", features = ["derive"] }
 serde_json = "1"
-clap = "=3.0.0-beta.2"
-clap_derive = "=3.0.0-beta.2"
+clap = { version = "3.0.0-rc.7", features = ["derive"] }
```

Comment:
This should be at least `^3.0.0`, I think?

Reply:
The version, which is in `Cargo.lock` matters. We use `3.0.12`.

I will change this to be `version = "3.0.0"` to be less confusing.